relays/table_transform.ts

```diff
@@ -0,0 +1,43 @@
+"use strict";
+
+import { Message } from "mhub";
+
+/**
+ * @param msg Message just received on this binding.
+ * @return Message(s), promise for message(s), or void
+ */

```

Comment:
This comment block belongs to the transform function, and is used to document the transform. Either remove it, or move it back to the function, below.